### ISSUE TITLE
documented setting `Ohai.plugin_path` in `client.rb` is not working with chef 12.16

### ISSUE THREAD

**Issue by sandervanburken** [chef/chef-web-docs#277]
Set this in `client.rb`:

```
Ohai.plugin_path << '/etc/chef/ohai/plugins'
```

According to the docs (https://docs.chef.io/config_rb_client.html under Ohai Settings) this should work. But it stops on a fatal error:

```
FATAL: Configuration error NoMethodError: undefined method plugin_path' for Ohai:Module
```

**Comment by webframp** [comment]
Looks like this will be fixed from changes in #249

**Comment by davidwrede** [comment]
PR #249 has been merged, so I'm closing out this issue.
